.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
       scala: 2.11.7
       env: TEST_SPARK_VERSION="1.5.0"
 script:
-  - sbt -Dspark.testVersion=$TEST_SPARK_VERSION ++$TRAVIS_SCALA_VERSION coverage test
```

Comment:
Can I get that back at another PR for that? I will open them right after closing this.